### ISSUE TITLE
Animationen

### ISSUE THREAD

**Issue by hurzl** [raphaelm/opacclient#328]
Es wird ziemlich viel Platz durch die Kästchen und noch mehr Zeit durch die Animation beim "Aufklappen" verbraucht. Und man muss jede Vormerkung einzeln aufmachen, um zu sehen, ob sie bereitgestellt ist.

**Comment by johan12345** [comment]
Dass die Karten mehr Platz verbrauchen als die alte, zweispaltige Darstellung, war uns bewusst. Dafür werden jetzt zum Beispiel Buchtitel nicht so leicht abgeschnitten, weil sie eben fast die ganze Breite zur Verfügung haben. Außerdem erkennt man durch die Icons vielleicht etwas besser, was die Textfelder bedeuten. Damit wir vertikal trotzdem nicht doppelt so viel Platz brauchen, haben wir den Ausklapp-Mechanismus eingebaut und zeigen dann in der nicht ausgeklappten Karte nur Autor und Titel (zusammen in einem Feld) und Status an.
Die Animation könnten wir schneller machen, ich weiß nicht, was @raphaelm dazu sagt.
Soweit ich die [Adis-API](https://github.com/raphaelm/opacclient/blob/master/opacclient/libopac/src/main/java/de/geeksfactory/opacclient/apis/Adis.java#L1277) und den Code des [AccountFragment](https://github.com/raphaelm/opacclient/blob/master/opacclient/opacapp/src/main/java/de/geeksfactory/opacclient/frontend/AccountFragment.java#L1179) verstehe, sollte bei bereitgestellten Bestellungen eigentlich der Text "bereit" im Status-Textfeld (das mit der Ampel) stehen, das man auch im nicht ausgeklappten Zustand sieht. Wenn nicht, ist das wohl ein Fehler in der API.

**Comment by hurzl** [comment]
Das "Bereit:" steht vor der Bibliothek, also in der 2. Zeile

**Comment by raphaelm** [comment]
@hurzl Steht bei dem "Bereit:" denn das Ampelsymbol dabei? Kannst du uns evtl. einen Screenshot schicken?

**Comment by hurzl** [comment]
Tja das Buch ist abgeholt, habe im Moment kein "Bereit"es mehr ...

**Comment by raphaelm** [comment]
Hat ja keine Eile :)

**Comment by hurzl** [comment]
Nein, eine Ampel ist es nicht, aber was es sein soll, kann ich auch nicht erkennen ;)

**Comment by raphaelm** [comment]
Dann wär ein Screenshot schon echt praktisch ;)

**Comment by hurzl** [comment]
echt. 
![screen](https://cloud.githubusercontent.com/assets/628746/9441467/93a55b64-4a76-11e5-8cc6-af8db7498ccf.png)

**Comment by johan12345** [comment]
Seit Version 5.0.0 haben wir eine neue Darstellung der Kontoansicht. Das Aufklappen wurde dabei durch das Öffnen eines Dialogs mit allen Details ersetzt und sollte nicht mehr ganz so lange dauern.

Was hier zu tun bleibt ist die Anzeige des "Bereit" an der richtigen Stelle (also unter "Status" statt "Zweigstelle"). @raphaelm, brauchen wir dafür noch irgendwas? Einen Screenshot der Ansicht auf der Bibliothekswebsite?

**Comment by raphaelm** [comment]
Ja, Screenshot von der Bibliothekswebsite wäre vermutlich praktisch. Auch wenn Adis da einfach nur einen Freitext hat, den Bibliotheksübergreifend gut zu parsen wird nie gehen.
